apps/go/requester/cmd/generate_sample.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+import (
+	"context"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"os"
+	"packages/mongodb"
+	"packages/utils"
+	"requester/types"
+	"requester/x"
+	"time"
+)
+
+func GetTask(node string) *types.Task {
+	return &types.Task{
+		Id: primitive.NewObjectID(),
+		RequesterArgs: types.RequesterArgs{
+			// found in localnet
+			Address: node,
+			Service: "0001",
+			Path:    "/v1/query/height",
+		},
+		Done: false,
+	}
+}
+
+func GetTasks(count int, node string) []*types.Task {
+	docs := make([]*types.Task, count)
+	for i := 0; i < count; i++ {
+		docs[i] = GetTask(node)
+	}
+	return docs
+}
+
+func GetInstance(task *types.Task) *types.Instance {
+	instance := types.Instance{
+		Id:   primitive.NewObjectID(),
+		Done: false,
+	}
+	if task != nil {
+		instance.TaskId = task.Id
+	}
+	return &instance
+}
+
+func GetInstances(count int, task *types.Task) []*types.Instance {
+	docs := make([]*types.Instance, count)
+	for i := 0; i < count; i++ {
+		docs[i] = GetInstance(task)
+	}
+	return docs
+}
+
+func GetPrompt(task *types.Task, instance *types.Instance) *types.Prompt {
+	prompt := types.Prompt{
+		Id: primitive.NewObjectID(),
+		// ethereum mainnet mock
+		Data:    "{}",
+		Timeout: 10000,
+		Done:    false,
+	}
+	if task != nil {
+		prompt.TaskId = task.Id
+	}
+	if instance != nil {
+		prompt.InstanceId = instance.Id
+	}
+	return &prompt
+}
+
+func GetPrompts(count int, task *types.Task, instance *types.Instance) []*types.Prompt {
+	prompts := make([]*types.Prompt, count)
+	for i := 0; i < count; i++ {
+		prompts[i] = GetPrompt(task, instance)
+	}
+	return prompts
+}
+
+func main() {
```

Comment:
Is this file a test or debug? do we need this here?

Reply:
thats only to generate data before sampler is done or to test relayer